### PR TITLE
Map seslhq.wip

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
Ok, 3rd Try at a PR from the correct place!

screenshot 2017-08-01 14 33 58

### CODE REVIEW

File: src/contentscripts/jobs/situational_awareness.js [hunk 1]
```diff
@@ -9,43 +9,75 @@ inject('jobs/situational_awareness.js');
...
+const rfscorpIcon = chrome.extension.getURL('icons/rfs.png');
+
 
 // Add the buttons for the extra layers
 $(<li id="lhlayers">
     <a href="#" class="js-sub-menu-toggle">
-      <img src={lighthouseIcon} style="width: 16px" />
-      <span class="text">Lighthouse</span>
-      <i class="toggle-icon fa fa-angle-left"></i>
+        <img src={lighthouseIcon} style="width:14px;vertical-align:top;margin-right:10px;float:left" />
```
Comment (sdunster) [inline src/contentscripts/jobs/situational_awareness.js]: this was perfectly fine without the extra 2 spaces :P

File: src/contentscripts/jobs/situational_awareness.js [hunk 2]
```diff
@@ -59,7 +91,7 @@ var timers = {};
  * @param interval the refresh interval.
  * @return timer the timer which refreshes the layer.
  */
-function registerClickHandler(buttonId, layer, updateFunction, interval) {
+ function registerClickHandler(buttonId, layer, updateFunction, interval) {
```
Comment (sdunster) [inline src/contentscripts/jobs/situational_awareness.js]: rly

File: src/injectscripts/jobs/situational_awareness.js [hunk 3]
```diff
@@ -281,6 +286,70 @@ const rfsIcons = {
...
+        for (let i = 0; i < data.features.length; i++) {
+            let hq = data.features[i];
+
+            let x = hq.geometry.x;
+            let y = hq.geometry.y;
+
+            let name = hq.attributes.HQNAME;
+
+            let details =
+            `<div id='lhqPopUp'>\
```
Comment (sdunster) [inline src/injectscripts/jobs/situational_awareness.js]: http://gph.is/1JqiRul

Comment (OSPFNeighbour) [inline src/injectscripts/jobs/situational_awareness.js]: you want JSX?

**Pull request merged**